.changeset/late-shoes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Adds border gradient and color theme.
```

Comment:
Revamp message slightly here for detail? This will be added to the changelog. Something like:

> Adds .border-color-<name> Atomic class, which applies a gradient border to a container.